var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -348,18 +361,45 @@ def cmake_args(self):
     def pre_install(self):
         with working_dir(self.build_directory):
             # When building shared libraries these need to be installed first
-            make('install-LLVMTableGen')
+            ninja('install-LLVMTableGen')
             if self.spec.version >= Version('4.0.0'):
                 # LLVMDemangle target was added in 4.0.0
-                make('install-LLVMDemangle')
-            make('install-LLVMSupport')
+                ninja('install-LLVMDemangle')
+            ninja('install-LLVMSupport')
 
     @run_after('install')
     def post_install(self):
+        spec = self.spec
+
+        # unnecessary if we get bootstrap builds in here
+        if '+cuda' in self.spec:
+            ompdir = 'build-bootstrapped-omp'
+            # rebuild libomptarget to get bytecode runtime library files
```

Comment:
In order to get the byte code built, you also need a:
`cmake_args.append('-DLIBOMPTARGET_NVPTX_ENABLE_BCLIB=true')`

Reply:
That shouldn't be required, it builds them if the compiler is right, but it's better to have the option there so it will error out if something goes wrong.  Added.